content/docs/dvclive/user-guide/ml-frameworks/pytorch.md

```diff
@@ -0,0 +1,60 @@
+# PyTorch
+
+The DVCLive - Pytorch integration allows you to easily add experiment tracking
```

Comment:
```suggestion
DVCLive allows you to easily add experiment tracking
```
I wouldn't call this an integration. It's really just a tutorial using pytorch. Same with TensorFlow. Open to other ideas for how to introduce these (or if they are necessary).

Reply:
I think that it sounds good the way you suggested